chrome-launcher/chrome-launcher.ts

```diff
@@ -142,7 +171,7 @@ export class ChromeLauncher {
     return spawnPromise.then(pid => Promise.all([pid, this.waitUntilReady()]));
```

Comment:
by profilePath do you mean `TMP_PROFILE_DIR` ?

I am actually think this was never working oddly enough..
https://github.com/GoogleChrome/lighthouse/search?utf8=%E2%9C%93&q=TMP_PROFILE_DIR&type=

Since I see no assignment anywhere to the value
--

thoughts?